package/MDAnalysis/coordinates/CRD.py

```diff
@@ -151,11 +154,24 @@ def __init__(self, filename, **kwargs):
         ----------
         filename : str or :class:`~MDAnalysis.lib.util.NamedStream`
              name of the output file or a stream
+
+        extended : bool (optional)
+             By default, noextended CRD format is used [``False``].
+             However, extended CRD format can be forced by
+             specifying `extended` ``=True``.
+
+
+        .. versionchanged:: 2.2.0
+           CRD extended format can now be explicitly requested with the
+           `extended` keyword
```

Comment:
You already have the version changed in the class docs. Perhaps we just mark the new argument as added here. Check if this formats correctly.

```suggestion

             .. versionadded:: 2.2.0
```